handy/logging.cc

```diff
@@ -76,8 +76,10 @@ void Logger::setFileName(const string& filename) {
 }
 
 void Logger::maybeRotate() {
+  //以第一次设置的日志文件名为日志基准文件名
+  static std::string basename = filename_;
```

Comment:
This local static variable may make readers confused.
Add some comments for it?

Reply:
I guess that the author's purpose is that rolling the log file after per logInterval_. However, the original function named with mayRotate can't finish it. So, i add the static variable to record the first filename_, meanwhile, make the value as a basename. The real log filename maybe constructed by basename and current time.

Reply:
Also, I meant add some comments in the line before `static .....` to explain why we use  local static variable here.

Reply:
Sorroy, I understand your suggestion wrong.  besides, I will finish it as soon as possible.

Reply:
The best way to solve this problem is changing class member filename_ to basename_, and change function setFileName(const std::string&) to setBaseName(const std::string&). However, the simplest way to meet the demand is adding a local static variable. It doesn't change the strcuture of Logger class.

Reply:
I think changing `filename_` to `basename_` will make more sense. 
We could do everything that make our codes better, right?

Reply:
I'm so sorry. After testing, i found the original function can work well.

Reply:
Fine. I will close this PR.